### PR TITLE
Remove spree store

### PR DESCRIPTION
**Pull request by mbj**

### CODE REVIEW

**Comment by mbj** [comment]
@dkubb This is ready for review.

File: app/mailers/spree/user_mailer.rb [hunk 1]
```diff
@@ -1,9 +1,17 @@
 module Spree
   class UserMailer < BaseMailer
-    def reset_password_instructions(user, token, *args)
-      @edit_password_reset_url = spree.edit_spree_user_password_url(:reset_password_token => token, :host => Spree::Store.current.url)
+    def reset_password_instructions(user, token, _opts = {})
```
Comment (dkubb) [inline app/mailers/spree/user_mailer.rb]: What about `**` instead of `_opts = {}`?

Comment (mbj) [inline app/mailers/spree/user_mailer.rb]: @dkubb I like that idea.

**Comment by mbj** [comment]
@dkubb Ready for the next pass.

**Pull request merged**